npm-shrinkwrap.json

```diff
@@ -11361,11 +11361,12 @@
 				"globby": "^9.0.0",
 				"hash.js": "^1.1.5",
 				"interpolate-components": "1.1.1",
-				"jed": "1.1.1",
 				"lodash": "^4.7.11",
 				"lru": "^3.1.0",
 				"moment": "^2.24.0",
 				"react": "^16.8.3",
+				"sprintf-js": "^1.1.2",
+				"tannin": "^1.1.0",
```

Comment:
It's been some time since I've worked with shrinkwrap (vs. `package-lock.json`), but should we expect to see some of the nested dependencies here? (e.g. `@tannin`-scoped packages `@tannin/compile`, etc)

Reply:
Looking at the main file for `tannin`, it does appear to rely on `@tannin/plural-forms`, which I would expect to be listed here too. I'm not sure why it wasn't picked up, but the build appears to work correctly. Odd.

Reply:
Actually, this appears to be correct! Here, only the direct dependencies for a package are listed. The sub-dependencies were already present int the file, likely due to Gutenberg. You can confirm by viewing the full file and searching for `@tannin`.